spec.md

```diff
@@ -534,9 +534,16 @@ Credentials are a set of information used by an application or a user to utilize
 
 #### Log Drain ####
 
-There are a class of service offerings that provide aggregation, indexing, and analysis of log data. To utilize these services an application that generates logs needs information for the location to which it will stream logs. If a broker represents one of these services, it MAY return a `syslog_drain_url` in the response for a request to create a service binding, to which logs MUST be streamed.
+There are a class of service offerings that provide aggregation, indexing, and
+analysis of log data. To utilize these services an application that generates
+logs needs information for the location to which it will stream logs. A
+create binding response from one of these services MUST include a
+`syslog_drain_url`. Applications MUST use this URL when sending logs to the
+service.
```

Comment:
>  I assume its ok for the app to send logs there too, right?

That's a platform choice whether require hosted apps to embed syslog libraries and have networking access to reach syslog drain endpoint. CF takes this burden off the apps and expects apps to be writing to stdout/stderr. 

Up to now the OSB was not normative about describing the usage of the syslog drain url:
- who is writing the logs (app vs platform)
- the expected format the log, see https://github.com/cloudfoundry/loggregator/issues/134
- the expected content/meta-data from the log (e.g. stdout vs stderr, timestamp)

Clarifying this would certainly help making syslog drain services more portable across platforms, and might deserve its own issue.